hw/net/rocker/rocker_world.c

```diff
@@ -67,6 +75,9 @@ void world_free(World *world)
     if (world->ops->uninit) {
         world->ops->uninit(world);
     }
+    if (world->name) {
```

Comment:
yes. will remove the check.

On Sun, Apr 19, 2015 at 9:22 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_world.c
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663420>:
>
> > @@ -67,6 +75,9 @@ void world_free(World *world)
> >      if (world->ops->uninit) {
> >          world->ops->uninit(world);
> >      }
> > +    if (world->name) {
>
> Seems world->name will always be non-NULL here, so no need to check for
> NULL.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663420>.
>